Nuro/Utilities/Constants.swift

```diff
@@ -6,12 +6,17 @@
 //
 
 import Foundation
+import UIKit
 
 /// Put any constant variables here like paths, etc
 
 class Constants {
```

Comment:
Dont throw all constant in 1 file. give a context such as Path  /Theme (For example)